components/scream/src/physics/share/tests/physics_universal_unit_tests.cpp

```diff
@@ -181,6 +214,11 @@ struct UnitWrap::UnitTest<D>::TestUniversal
         Real zi_bot = (pow(k,2)+k)/2.0;
         Real zi_top = zi_bot + (k+1); 
         dz_tests(zi_top,zi_bot,errors);
+        // DSE Test
+        Real temp = k+1;
+        Real height = k+1;
+        Real surface_height = k+1;
+        dse_tests(temp,height,surface_height,errors);
```

Comment:
Not sure what values make sense for the test.. They just need to be >0 to pass all checks..

Reply:
Why not using random numbers in something like [0.001,1000.0]? You can use `ekat::genRandArray(&temp,1,engine,pdf)`. See, e.g., [field tests](https://github.com/E3SM-Project/scream/blob/440e098e9a16ee5feb41a68c56fee69d9f6dfecc/components/scream/src/share/tests/field_tests.cpp#L493), which also shows how to give the ability to pass the rng seed to the test exec (in case you're debugging and want to match someone else's buggy numbers).